services/app-api/handlers/measures/get.ts

```diff
@@ -35,16 +33,16 @@ export const listMeasures = handler(async (event, context) => {
     }
   } // if not state user, can safely assume admin type user due to baseline handler protections
   const params = {
-    TableName: process.env.measureTableName!,
-    ...convertToDynamoExpression(
-      { state: state, year: parseInt(year), coreSet: coreSet },
-      "list"
-    ),
+    TableName: process.env.measureTable!,
+    KeyConditionExpression: "compoundKey = :compoundKey",
+    ExpressionAttributeValues: {
+      ":compoundKey": `${state}${year}${coreSet}`,
+    },
   };
 
-  let queriedMeasures = await dynamoDb.scanAll<Measure>(params);
+  const queriedMeasures = await dynamoDb.queryAll<Measure>(params);
   for (let v of queriedMeasures) {
-    const measure = measures[parseInt(year as string)]?.filter(
+    const measure = measures[year as number]?.filter(
```

Comment:
Good news, this cast is no longer necessary!

```suggestion
    const measure = measures[year]?.filter(
```

Reply:
we just chatted about this but for anyone who is curious, typescript is an idiot and unfortunately needs this godforsaken cast even though IT KNOWS year is a mf number.


just a note that if you're looking at this in vscode, it won't throw the error inline, it will only fail during the tests because of a ts mismatch between vscode and qmr. or something